Common/Inc/CommonDataTypes.hpp

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Gordon Fountain 2022-11-30.
+//
+
+#ifndef ZPSW3_COMMON_DATATYPES_HPP
+#define ZPSW3_COMMON_DATATYPES_HPP
+
+// Struct copied from old code
+typedef struct {
+    float roll, pitch, yaw; //rad
+    float rollRate, pitchRate, yawRate; //rad/s
+    float airspeed; //m/s
+    float altitude; //m
+    float rateOfClimb; //m/s
+    long double latitude; //Decimal degrees
+    float latitudeSpeed; //m/s
+    long double longitude; //Decimal degrees
+    float longitudeSpeed; //m/s
+    double track; // degrees
+    double heading; //degrees
+} SFOutput_t;
```

Comment:
I think this needs to be renamed to LosSFData

Reply:
Resolved in 5af498c